testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/vault/KeycloakVaultTest.java

```diff
@@ -81,6 +81,9 @@ public void run(KeycloakSession session) {
             try (VaultStringSecret secret = transcriber.getStringSecret(invalidEntry)) {
                 Optional<String> optional = secret.get();
                 Assert.assertFalse(optional.isPresent());
+            } catch (RuntimeException e) {
+                // expected exception is thrown.
+                Assert.assertEquals("Cannot invoke \"java.security.Key.getEncoded()\" because \"key\" is null", e.getMessage());
```

Comment:
I think  we should remove this now that the FIPS issues are resolved.

Reply:
This has nothing to do with FIPS though. It tests invalid entry in the vault.

Reply:
Then we should probably have some switch or something to expect the exception only when relevant.

Reply:
Do you think that this is still relevant? (see my previous comment)